schemas/com.snowplowanalytics.snowplow/cross_navigation/jsonschema/1-0-0

```diff
@@ -0,0 +1,71 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a user and session identifiers entity included in cross platform or cross domain navigation.",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "cross_navigation",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "user_id": {
+            "description": "The business user ID, the user was identified in the source app.",
+            "type": [
+                "null",
+                "string"
+            ],
+            "maxLength": 4096
+        },
+        "domain_user_id": {
+            "type": "string",
+            "maxLength": 36,
+            "description": "Domain user ID of the source app from which a link was followed."
+        },
+        "session_id": {
+            "type": [
+                "null",
+                "string"
+            ],
+            "format": "uuid",
+            "description": "Session ID of the source app from which a link was followed."
+        },
+        "timestamp": {
```

Comment:
The corresponding atomic property is named `refr_dvce_tstamp`. We don't have to use the same name, but wondering whether being consistent has some advantage. In addition, for this case in particular the atomic name seems a bit more descriptive than just `timestamp`.